trace-server/org.eclipse.tracecompass.incubator.trace.server.jersey.rest.core/src/org/eclipse/tracecompass/incubator/internal/trace/server/jersey/rest/core/services/ServerInfoResponseImpl.java

```diff
@@ -0,0 +1,152 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Ericsson
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License 2.0 which
+ * accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.tracecompass.incubator.internal.trace.server.jersey.rest.core.services;
+
+import org.eclipse.tracecompass.incubator.internal.trace.server.jersey.rest.core.model.ServerInfoResponse;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+/**
+ * Implementation of the ServerInfoResponse interface
+ *
+ * @author Vlad Arama
+ */
+public class ServerInfoResponseImpl implements ServerInfoResponse {
```

Comment:
I get a warning that this class implements an API that is not publicized (in MANIFEST.MF). That's on purpose because we don't want these classes to be implemented by downstream projects because they are just documentation. To keep that also in the future, I suggest to remove `implements ServerInfoRsponse`. You will have to copy the javadoc then and remove `@Overrides`.